src/bin/pg_dump/pg_dump.c

```diff
@@ -413,6 +416,7 @@ main(int argc, char **argv)
 		{"on-conflict-do-nothing", no_argument, &dopt.do_nothing, 1},
 		{"rows-per-insert", required_argument, NULL, 10},
 		{"include-foreign-data", required_argument, NULL, 11},
+		{"bbf-database-name", required_argument, NULL, 12},
```

Comment:
this is problematic as community can use 12 in future and we have to resolve it every time. May be use a large number to leave sufficient buffer for community

Reply:
Changed it 30 which I think should be good since it's very unlikely for dump utility to get 19 more options.